README.md

```diff
@@ -6,11 +6,13 @@
 To install the package, clone the repository and run:
 ```bash
 pip install -e .
+charge-install
```

Comment:
@bvanessen This returns:

```
Usage: charge-install [OPTIONS]
Try 'charge-install --help' for help.

Error: Invalid value for '--extras': Value must be an iterable.
```

Reply:
I tried with 
```
charge-install --extras all
```
And it worked. We could have `all` as the default but alternatively, we could also just change the doc. Both are reasonable IMO.